setup_flashgg.sh

```diff
@@ -76,6 +76,14 @@ cp XGBoostCMSSW/XGBoostInterface/toolbox/*xml $CMSSW_BASE/config/toolbox/$SCRAM_
 scram setup rabit
 scram setup xgboost
 
+echo "setting up ONNXRuntime"
```

Comment:
As the flashgg framework will move on CMSSW_10_6_29 for the Summer20UL campaign, the ONNX interface for ML no longer needs to download manually. Once the flashgg is updated, I will let you know and you can delete these.

Reply:
Ok, thanks a lot! Indeed, this should be removed when we switch to the CMSSW_10_6_29 version.